packages/@react-spectrum/toast/src/Toast.tsx

```diff
@@ -88,6 +88,22 @@ export const Toast = React.forwardRef(function Toast(props: SpectrumToastProps,
     }
   };
 
+  let wrappedCloseButtonProps = {
+    ...closeButtonProps,
+    onPress: (e) => {
+      if ('startViewTransition' in document) {
+        // @ts-ignore
```

Comment:
looks like this is included in TS ^5.6
we can probably use ts-expect-error so we know to remove it when we upgrade